lighthouse-core/test/scripts/i18n/collect-strings-test.js

```diff
@@ -79,34 +76,60 @@ describe('Compute Description', () => {
       /** 
        * @description Tagged description for Hello World.
        */
-      message: 'Hello World',
+      exampleString: 'Hello World',
     };`;
+    const liveUIStrings = evalJustUIStrings(justUIStrings);
+    const res = collect.parseUIStrings(justUIStrings, liveUIStrings);
 
-    const ast = esprima.parse(justUIStrings, {comment: true, range: true});
-
-    const stmt = ast.body[0];
-    const prop = stmt.declarations[0].init.properties[0];
-    const res = collect.computeDescription(ast, prop, 'Hello World', 0);
-    expect(res.description).toBe('Tagged description for Hello World.');
+    expect(res).toEqual({
+      exampleString: {
+        message: 'Hello World',
+        description: 'Tagged description for Hello World.',
+        examples: {},
+      },
+    });
   });
 
-  it.skip('collects complex multi-line description', () => {
+  it('collects complex multi-line description', () => {
```

Comment:
:heart: these tests.  Can we get one with a multi-line description and an example too?

Reply:
> Can we get one with a multi-line description and an example too?

done